Formula/objc-run.rb

```diff
@@ -10,18 +10,19 @@ class ObjcRun < Formula
     sha256 cellar: :any_skip_relocation, all: "e779505f8071d158730517e91004a15c2364dbf03acceebd1643e27338792f98"
   end
 
+  pour_bottle? do
+    reason "The bottle needs to be installed into #{Homebrew::DEFAULT_PREFIX}."
+    # https://github.com/Homebrew/homebrew-core/pull/76633
+    # Remove when the following issue is resolved:
+    # https://github.com/Homebrew/brew/issues/11302
+    satisfy { HOMEBREW_PREFIX.to_s == Homebrew::DEFAULT_PREFIX } unless Hardware::CPU.arm?
```

Comment:
Since we are using an `all` bottle, won't it already have `@@HOMEBREW_PREFIX@@`, and thus be replaced `/opt/homebrew` on ARM?

Reply:
No, it won't. The reason for the original `inreplace`

https://github.com/Homebrew/homebrew-core/blob/7dd76d8a800c73cb0a8e1488b7863b445158fa68/Formula/objc-run.rb#L15

was because there was an `@@HOMEBREW_PREFIX@@` on Intel, but not on ARM, causing the checksums to differ.

Reply:
Or rather, it won't anymore after rebuilding the bottles.

Reply:
Oh right, if we're rebuilding bottles then :+1: